UnitaryTesting/TestDemeritPointsCalculator.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ExercisesMoshCourse;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitaryTesting
```

Comment:
A standard practice is to name the testing project like... `{ProjectName}.Tests` so in your case could be... `ExercisesMoshCourse.Tests`

Reply:
I'll keep that in mind for the next exercises, thank you.